src/main/java/seedu/address/storage/JsonAdaptedEvent.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.storage;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.EventName;
+import seedu.address.model.event.EventTime;
+import seedu.address.model.module.ModuleName;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Jackson-friendly version of {@link Event}.
+ */
+public class JsonAdaptedEvent {
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Module's %s field is missing!";
```

Comment:
Module is referenced here instead of event.

Reply:
Will implement later on!